x-pack/plugin/ml/qa/ml-with-security/build.gradle

```diff
@@ -256,4 +257,5 @@ testClusters.configureEach {
   user username: "no_ml", password: "x-pack-test-password", role: "minimal"
   setting 'xpack.license.self_generated.type', 'trial'
   setting 'xpack.security.enabled', 'true'
+  requiresFeature 'es.inference_rescorer_feature_flag_enabled', Version.fromString("8.9.0")
```

Comment:
should this be 8.10 version?

Reply:
yes, I need to go through and update the version :)